src/components/DiffView/index.tsx

```diff
@@ -365,8 +367,11 @@ export class DiffViewBase extends React.Component<Props> {
     if (diff) {
       hunksToDisplay = diff.hunks;
 
-      if (getChangeCharCount(hunksToDisplay) >= _slowLoadingCharCount) {
-        if (!shouldAllowSlowPages({ allowByDefault: true, location })) {
+      if (
+        getChangeCharCount(hunksToDisplay) >= _slowLoadingCharCount ||
+        isMinified
```

Comment:
can we unify this check somewhere (either with a function or with a variable) so it's not defined in both `DiffView` and `CodeView`?

Reply:
Yes, I could do that. Thanks for the suggestion.